### PR TITLE
SC-9740: Adding PAXG-A

### PR DESCRIPTION
**Pull request by hexonaut**
# Description

https://app.clubhouse.io/scdomaincommunityteam/story/9740/deploy-paxg-to-kovan

# Contribution Checklist

- [x] First commit title starts with COB Team Name and Collateral type (e.g. `___-TOKEN-X`)
- [x] Code approved
- [x] Tests approved
- [x] CI Tests pass

# Checklist

- [x] Every contract variable/method declared as public/external private/internal
- [x] Verify expiration (`4 days + 2 hours` monthly and `30 days` for the rest)
- [x] Verify hash in the description matches [here](https://emn178.github.io/online-tools/keccak_256.html)
- [x] Validate all addresses used are in Kovan changelog
- [x] Deploy spell to kovan `SOLC_FLAGS="--optimize --optimize-runs=1" dapp --use solc:0.6.11 build --extract && dapp create DssSpell --gas=5000000 --gas-price="$(seth --to-wei 3 "gwei")"`
- [x] Ensure contract is verified on `kovan` etherscan
- [x] Change test to use kovan spell address and deploy timestamp
- [x] Keep `Kovan-DssSpell.sol` and `Kovan-DssSpell.t.sol` the same, but make a copy in `archive`
- [ ] `squash and merge` this PR

### CODE REVIEW

File: src/Kovan-DssSpell.sol [hunk 1]
```diff
@@ -33,45 +33,49 @@ contract DssSpellAction is DssAction {
...
-        DssExecLib.setChangelogAddress("MCD_JOIN_ETH_C", MCD_JOIN_ETH_C);
-        DssExecLib.setChangelogAddress("MCD_FLIP_ETH_C", MCD_FLIP_ETH_C);
+        DssExecLib.setChangelogAddress("PAXG", PAXG);
+        DssExecLib.setChangelogAddress("MCD_JOIN_PAXG_A", MCD_JOIN_PAXG_A);
+        DssExecLib.setChangelogAddress("MCD_FLIP_PAXG_A", MCD_FLIP_PAXG_A);
+        DssExecLib.setChangelogAddress("PIP_PAXG", PIP_PAXG);
 
         // bump changelog version
-        DssExecLib.setChangelogVersion("1.2.10");
+        DssExecLib.setChangelogVersion("1.2.11");
```
Comment (brianmcmichael) [inline src/Kovan-DssSpell.sol:78]: Let's not bump the version for now.

File: src/Kovan-DssSpell.sol [hunk 2]
```diff
@@ -33,45 +35,52 @@ contract DssSpellAction is DssAction {
...
+    // PAXG-A
+    address constant PAXG = 0x016e5fD2dDc4156Eb560E479b3F19b563B2e96B9;
+    address constant MCD_JOIN_PAXG_A = 0xBaBc2d08BA653d394ed36AE7a7720ac3FC43C954;
+    address constant MCD_FLIP_PAXG_A = 0xf0AC5f16535ed2f9c537420321E063C8d677a336;
+    address constant PIP_PAXG = 0x31CceDBc45179f17CfD34967680C6560b6509C1A;
+
+    address constant PROXY_ACTIONS = 0xD8b9702755E91Aa792656966aE6bAF32F4C394Ba;
 
     function actions() public override {
         // Onboarding ETH-C
```
Comment (brianmcmichael) [inline src/Kovan-DssSpell.sol:53]: delete comment

**Review by brianmcmichael — APPROVED** [review]
LGTM

**Comment by hexonaut** [comment]
Added archive and cast spell. Good to merge with final approval @brianmcmichael .

**Pull request merged**